### ISSUE TITLE
Translate/make cryptris documents in english/dutch

### ISSUE THREAD

**Issue by cr-marcstevens** [cr-marcstevens/cryptris#3]

**Comment by lducas** [comment]
Because external resources may vary from one language to another, I'll just create a copy of that page, and try to make sure your script ignores it :p

**Comment by cr-marcstevens** [comment]
The easiest is to just create a `documentation_en.html` in the root, but be careful: translate.sh will delete all `*_*.html` files, so that needs to be uncommented.

**Comment by lducas** [comment]
I created that file in a subdirectory, and the translate script does an exception to fetch that file. You can close issue if you are satisfied.

**Comment by cr-marcstevens** [comment]
The external doc links and pre-translated files are fine (just made it a bit more generic and fixed some typos).

Are you planning on having the cryptris docs translated? Because then I'll keep it open.

**Comment by lducas** [comment]
No, those texts are way too long I think. There may be a new document, inspired by them, but more in the 1-2 pages range.
